patches/@popperjs+core+2.9.2.dev.patch

```diff
@@ -1,91 +0,0 @@
-diff --git a/node_modules/@popperjs/core/lib/modifiers/preventOverflow.js b/node_modules/@popperjs/core/lib/modifiers/preventOverflow.js
-index 1776c09..e08aad1 100644
---- a/node_modules/@popperjs/core/lib/modifiers/preventOverflow.js
-+++ b/node_modules/@popperjs/core/lib/modifiers/preventOverflow.js
-@@ -52,26 +52,27 @@ function preventOverflow(_ref) {
-     return;
-   }
- 
--  if (checkMainAxis || checkAltAxis) {
--    var mainSide = mainAxis === 'y' ? top : left;
--    var altSide = mainAxis === 'y' ? bottom : right;
--    var len = mainAxis === 'y' ? 'height' : 'width';
--    var offset = popperOffsets[mainAxis];
--    var min = popperOffsets[mainAxis] + overflow[mainSide];
--    var max = popperOffsets[mainAxis] - overflow[altSide];
-+  function setOffset(axis) {
-+    var isMainAxis = axis === mainAxis;
-+    var mainSide = axis === 'y' ? top : left;
```

Comment:
Why did we need to edit the patch here?

Reply:
This popper js is the bundled one from `react-popper`. As we don't need it any more, I removed it and added new popper 2.11.6. So, we don't need the patch for 2.9.2 version.